services/streammanager/kafka/kafkamanager.go

```diff
@@ -234,6 +235,19 @@ func Init() {
 		)
 	}
 
+	kafkaCompression = client.CompressionZstd
+	if kc := config.GetInt("Router.kafkaCompression", -1); kc != -1 {
```

Comment:
maybe we should use the "standard" naming pattern, e.g. `Router.KAFKA.compression`

Reply:
I had exactly the same thought when I was coming up with the name but then I realized that the batching parameter is the only one following that pattern :sweat_smile: 

The majority of the parameters use this other one so I went with the majority. Not strong on this though, I can change it if you guys prefer it :+1: